lib/view_model/restore/restore_wallet.dart

```diff
@@ -46,12 +48,14 @@ class RestoredWallet {
     final height = json['height'] as String?;
     final mnemonic_seed = json['mnemonic_seed'] as String?;
     final seed = json['seed'] as String? ?? json['hexSeed'] as String?;
+    final passphrase = json['passphrase'] as String?;
```

Comment:
ok good, but we should also add it to the wallet QR that Cake generates in the seeds/keys page so that the QR that cake generates would work as expected